youtube_dl/extractor/peertube.py

```diff
@@ -513,6 +520,13 @@ def _real_extract(self, url):
             formats.append(f)
         self._sort_formats(formats)
 
+        subtitles = {}
+        for entry in video_captions['data']:
+            caption_url = 'https://%s%s' % (host, entry['captionPath'])
+            subtitles[entry['language']['id']] = [{
```

Comment:
Does not handle multiple subtitle formats per language scenario.

Reply:
PeerTube does not currently enable this scenario, as only one subtitle file may exist for each language. Should I still change this code?

Reply:
You should as already pointed out.